### PR TITLE
refactor: #id 22731 Wrap yes/no dialog and move it to finsemble UI repo

### PR DESCRIPTION
**Pull request by zona-ciq**
refactor: #id 22731

[Kanban link](https://chartiq.kanbanize.com/ctrl_board/18/cards/22731/details/)

**Description of change**
* Move the yesNoDialog code from finsemble seed to finsemble ui

**Description of testing**
1. Launch finsemble, try to switch to another workspace when the current workspace is dirty. When the yes/no Dialog pops up make sure it is showing the correct content and is working as before.

### CODE REVIEW

File: build/webpack/webpack.finsemble-built-in.entries.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	},
 	"yesNoDialog": {
 		"output": "components/yesNoDialog/yesNoDialog",
-		"entry": "./src-built-in/components/yesNoDialog/src/app.jsx"
+		"entry": "./src-built-in/components/yesNoDialog/yesNoDialog.jsx"
```
Comment (d4hines) [inline build/webpack/webpack.finsemble-built-in.entries.json]: The other components have a file called "app.jsx". Why change it?

Comment (zona-ciq) [reply build/webpack/webpack.finsemble-built-in.entries.json]: Just to be more clear - to void having a bunch of app.jsx files to sort through when you search for a file

Comment (d4hines) [reply build/webpack/webpack.finsemble-built-in.entries.json]: I totally agree with you, but we should be consistent throughout the whole app as a matter of polish. Maybe we can make a quick card to fix all of these.

Comment (zona-ciq) [reply build/webpack/webpack.finsemble-built-in.entries.json]: Yup. I created a card to address that.

**Pull request merged**